landing_page/app/javascript/components/Microposts.test.js

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
General comment about all your tests files - generally test files do not live with the source code they are testing and instead live in a separate directory.  Eg for portal see the 'app/webpack/components/seamless_login_access' dir. there is a sperate __tests__ folder.